application/app/src/test/java/uk/nhs/digital/uec/api/controller/FuzzySearchControllerTest.java

```diff
@@ -83,7 +83,8 @@ public void getServicesByFuzzySearchTestSucc() throws ValidationException {
             NAME_PRIORITY,
             ADDRESS_PRIORITY,
             POSTCODE_PRIORITY,
-            PUBLIC_NAME_PRIORITY);
+            PUBLIC_NAME_PRIORITY,
+            null);
```

Comment:
Collections.EmptyMap

Reply:
These changes were already merged, this branch was rebased from master